docs/API_SPECS_SIMULATIONS.md

```diff
@@ -611,10 +611,45 @@ to ignore any change happened and overwrite the existing simulation.  Note
 that this is meant to be used only when data loss is acceptable, e.g.
 during development sessions and in test environments.
 
-## Deleting a simulation
+## Delete a simulation
 
 Simulations can be deleted using the DELETE method.
 
 ```
 DELETE /v1/simulations/1
-```
\ No newline at end of file
+```
+
+## Add device in a simulation
+
+Add a device to a simulatiom using AddDevice method.
+
+Request:
+```
+PUT /v1/simulations/1/AddDevice
```

Comment:
this URL is not consistent with other endpoints and guidelines, please avoid simple verbs (add) that can be translated to HTTP verbs (POST and PUT). Suggested:
* PUT /v1/devices/{id} and simulation ID in the payload
* POST /v1/devices and simulation ID in the payload, device ID is randomly generated
alternatively, but discouraged because devices can be shared:
* /simulations/{sim id}/devices/{device id}